README.md

```diff
@@ -497,140 +497,3 @@ Available controllers:
 
 Moving the robot:
   - see below description of `forward_position_controller`
-
```

Comment:
I would also add into the main readme that example of controller switching is under example 1. What do you think?